src/GovUk.Education.ExploreEducationStatistics.Public.Data.Model/DataSetVersionImport.cs

```diff
@@ -0,0 +1,28 @@
+using GovUk.Education.ExploreEducationStatistics.Common.Model;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Model;
+
+public class DataSetVersionImport : ICreatedUpdatedTimestamps<DateTimeOffset, DateTimeOffset?>
+{
+    public Guid Id { get; init; }
+
+    public required Guid DataSetVersionId { get; set; }
+
+    public DataSetVersion DataSetVersion { get; set; } = null!;
+
+    public required DataSetVersionImportStatus Status { get; set; }
```

Comment:
I didn't add a separate `Stage` just yet because I thought this could be added as we develop the importer further and also because I have a question over how much detail we need to persist here vs. what detail can we query from the function itself to get the current status of function.